### ISSUE TITLE
Interactive Previews

### ISSUE THREAD

**Issue by mjhenkes** [cerner/kaiju#9]
Update code generation to allow users to build interactive previews. For starters we should focus on modal and popup disclosures.

**Comment by claytonbeese** [comment]
Is this still on the docket to work on? It would be super helpful for projects I've been working on

**Comment by mjhenkes** [comment]
Yes, but it's a big project. We plan to start work on it in 2019.

**Comment by tm4027** [comment]
•Enhance 	Performance
o	Improve performance of common actions.
o	Many actions within Kaiju are slow. These actions become even slower for larger workspaces.
o	Common Actions
	Loading Workspaces
	Adding components to a large workspace
	Deleting components from a large workspace
	Dragging components within the layers panel.
	Deleting Projects
	Deleting Workspaces
